tools/accuracy_checker/configs/handwritten-japanese-recognition-0001.yml

```diff
@@ -0,0 +1,40 @@
+models:
+  - name: handwritten-japanese-recognition-0001
+
+    launchers:
+      - framework: dlsdk
+        device: cpu
+        tags:
+          - FP16
+        model:   intel/handwritten-japanese-recognition-0001.xml
+        weights: intel/handwritten-japanese-recognition-0001.bin
+        adapter: ctc_greedy_search_decoder
+
+    datasets:
+      - name: Kondate
+        # In order to be used by model, images must be:
+        # 1) Resized to fixed height 96, using AREA interpolation
+        # 2) Normalized by mean 127.5 and std 127.5
+        # 3) Padding with edge padding, right padding only
+        preprocessing:
+          - type: bgr_to_gray
+          - type: resize
+            interpolation: AREA
+            aspect_ratio_scale: width
+            size: 96
+          - type: normalization
```

Comment:
usually, we place normalization to model IR during MO conversion

Reply:
moreover, for intel models it is requirement